RLTest/__main__.py

```diff
@@ -208,7 +208,7 @@ def do_normal_conn(self, line):
 parser.add_argument(
     '--vg-suppressions', default=None, help='path valgrind suppressions file')
 parser.add_argument(
-    '--vg-options', default='--leak-check=full --errors-for-leak-kinds=definite', dest='vg_options', help='valgrind [options]')
+    '--vg-options', default=None, dest='vg_options', help='valgrind [options]')
```

Comment:
Why changing the default?

Reply:
The command line option always overrides the env var. I've changed the default to None so I can detect whether the command line option was provided, and if not, probe for the env var. If the latter is missing, the old default is taken.